Kerberos.NET/Client/Krb5TicketCache.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Kerberos.NET.Client
+{
+    [DebuggerDisplay("{cache}")]
+    public class Krb5TicketCache : TicketCacheBase
+    {
+        private readonly string filePath;
+        private readonly Krb5CredentialCache cache;
+
+        private readonly object fileSync = new object();
```

Comment:
```suggestion
        private readonly string _filePath;
        private readonly Krb5CredentialCache _cache;

        private readonly object _fileSync = new object();
```
According the naming guidelines in dotnet, the `_` prefix should be used for fields.

Reply:
We will be adopting a different guideline which is used internally by the team, which requires not using them.